OutOfSchool/OutOfSchool.WebApi.Tests/Controllers/ChildControllerTests.cs

```diff
@@ -23,6 +23,8 @@ public class ChildControllerTests
     {
```

Comment:
Please add constructor tests. (Assert that it fails with null passed as any service parameter)